yt_dlp/extractor/arte.py

```diff
@@ -174,7 +174,7 @@ def _real_extract(self, url):
         return {
             'id': player_info.get('VID') or video_id,
             'title': title,
-            'description': player_info.get('VDE'),
+            'description': player_info.get('VDE') or player_info.get('V7T'),
```

Comment:
could there be other possiblities? Maybe it is better to do a regex match (eg: `V\w{2}`) on the keys

Reply:
@pukkandan thanks for the advice ! :-) No other possibilities, I guess.
V7T is always filled. If VDE (full description) is empty, V7T will be take in account.
I could try the regex, but frankly I'm not an regex expert :-)